### ISSUE TITLE
train & test split

### ISSUE THREAD

**Issue by wuy202** [fightingMinty/WHU-road-dataset#1]
In the BT-RoadNet paper, you mentioned the WHU-road-dataset were split into 5121 training samples and 1707 for testing. Would you provide the split file?

**Comment by fightingMinty** [comment]
Sorry, we lost the file list of the training set and test set in the machine update. You can randomly divide the training set and test set in proportion.
